### ISSUE TITLE
Issues with password reset

### ISSUE THREAD

**Issue by Eitriana** [ThanokKarabor/Karabor#10]
Initial error:
After following password reset link in email (using Gmail, Chrome browser) and entering in new 12 character password, received error stating reset link is invalid. Screenshot shown here:

`http://imgur.com/a/Pk7sr`

Attempted to follow link on IE, same result.
Attempted to follow link on mobile Chrome browser connected to Wifi, same result.
Attempted to follow link on mobile Chrome browser off Wifi, same result.

Admin could follow link and change password with no issues.

**Comment by ThanokKarabor** [comment]
Yeah this is broken for the moment, having issues with bind_param on an SQL statement :(

Here's the code if anyone can figure out why it's returning NULL I'd be super excited!

$sql = "UPDATE account SET username=?, sha_pass_hash=?, email=? WHERE username=?"; 
if($stmt = $conn->prepare($sql)) {
//die($username.":".$password.":".$email);
$stmt->bind_param("ssss",$username,$password,$email,$username);
$stmt->execute();
if ($conn->query($sql) === TRUE) {
     die("Record updated successfully");
} else {
     die("Error updating record: " . $conn->error);
}
}else{
     var_dump($conn->db->error);
}`
